src/tests/Microsoft.PowerFx.Interpreter.Tests/InterpreterSuggestTests.cs

```diff
@@ -22,7 +22,7 @@ private string[] SuggestStrings(string expression, PowerFxConfig config, RecordT
         {
             Assert.NotNull(expression);
 
-            var intellisense = Suggest(expression, config, parameterType);
+            var intellisense = Suggest(expression, config, null, parameterType);
```

Comment:
>null [](http://example.com/codeflow?start=59&length=4)

[minor] can you use a named argument (culture: null)?

Reply:
good idea, it's clearer with culture: